### DISCUSSION TITLE
Profile is not showing in repo

### DISCUSSION THREAD

**Discussion by AbhinavTheDev** [EddieHubCommunity/LinkFree#8323] [Q&A] [answered]
Hello Folks,

I want to know that I create my linkfree profile with forms so i try to find my profile name in data directory of repo but it is not available there . Do i need to made profile with json or is it available in any other directory. please tell me.

Also tell me which method is better for creating profile Forms or JSON.

**Comment by eddiejaoude** [comment, marked as answer]
There are 2 ways to manage your profile, either json file or forms. If you use forms then there will be no json file in the data directory.

Forms are the recommended way now because they get the new features. 

I can see your profile is live here https://linkfree.io/Abhinav11234

**Reply by AbhinavTheDev** [reply]
thanks for your reply eddie but i think giving new features to only forms method users is not right for all us .We need to integrate JSON and Form method users So that both of them get most out from the linkfree.

Once Again Thanks For Reply Eddie
